File: src/main/org/nlogo/prim/etc/_clearlinks.java
package org.nlogo.prim.etc ;

import org.nlogo.nvm.Command;
import org.nlogo.nvm.Context;
import org.nlogo.nvm.Syntax;

public final strictfp class _clearlinks
	extends Command
{
	@Override
	public Syntax syntax()
	{
		return Syntax.commandSyntax( "O---" , true ) ;
	}
	@Override
	public void perform( final Context context )
	{
		world.clearLinks();
		context.ip = next ;
	}
}

File: src/main/org/nlogo/prim/etc/_clearlinks.scala
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.prim.etc

import org.nlogo.api.Syntax
import org.nlogo.nvm.{ Command, Context }

class _clearlinks extends Command {
  override def syntax =
    Syntax.commandSyntax("O---", true)
  override def perform(context: Context) {
    world.clearLinks()
    context.ip = next
  }
}
